# HG changeset patch
# User Sukhbir Singh <sukhbir@torproject.org>
# Date 1413961999 -19800
#      Wed Oct 22 12:43:19 2014 +0530
# Node ID bdb1f40bcd898f205de1f67b6947a9c77bbea651
# Parent  ce8c62cb7b7c24bb68fc5d16eba25b3f43c30786
Ensure TLS/SSL is used by default for IRC accounts

diff --git a/chat/protocols/irc/irc.js b/chat/protocols/irc/irc.js
--- a/chat/protocols/irc/irc.js
+++ b/chat/protocols/irc/irc.js
@@ -1808,8 +1808,8 @@
 
   options: {
     // TODO Default to IRC over SSL.
-    "port": {get label() _("options.port"), default: 6667},
-    "ssl": {get label() _("options.ssl"), default: false},
+    "port": {get label() _("options.port"), default: 6697},
+    "ssl": {get label() _("options.ssl"), default: true},
     // TODO We should attempt to auto-detect encoding instead.
     "encoding": {get label() _("options.encoding"), default: "UTF-8"},
     "quitmsg": {get label() _("options.quitMessage"),
